packages/main/src/themes/CalendarLegendItem.css

```diff
@@ -0,0 +1,73 @@
+.ui5-calendar-legend-item-root {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	width: var(--_ui5-calendar-legend-item-root-width);
+	vertical-align: middle;
```

Comment:
I think you don't need vertical-align: middle; here, and vertical-align: top; and vertical-align: middle; in the selectors a bit downwards